tdrs-frontend/src/components/GovBanner/GovBanner.jsx

```diff
@@ -26,28 +26,21 @@ function GovBanner() {
                   switch (process.env.NODE_ENV) {
                     case 'development':
                       return ' Development Demo '
-                      break
                     case 'test':
                       return ' Test Demo '
-                      break
```

Comment:
These were introducing a runtime linter failure, hence their removal. We will not re-add these.

@riatzukiza